### PR TITLE
Dev branch

### PR DESCRIPTION
**Pull request by deepakkujena**
Please review

### CODE REVIEW

File: Tes2.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+
+/**
+Test2 sample
+*/
+public class Test2{
```
Comment (Smjena) [inline Tes2.java]: class name and file name not matching

Comment (deepakkujena) [reply Tes2.java]: Will correct and check in

**Comment by Smjena** [comment]
fine now. go ahead formerge

**Comment by deepakkujena** [comment]
closing this pull request

**Pull request merged**